main/webapp/modules/core/scripts/views/data-table/menu-reconcile.js

```diff
@@ -55,6 +55,32 @@ DataTableColumnHeaderUI.extendMenu(function(column, columnHeaderUI, menu) {
     );
   };
 
+  var doAddColumnWithUrlOfMatchedEntities=function(){
+    var serviceUrl = null;
+    var service = null;
+    if (column.reconConfig) {
+        serviceUrl = column.reconConfig.service;
+    }
+    if (serviceUrl) {
+        service = ReconciliationManager.getServiceFromUrl(serviceUrl);
+    } 
+    Refine.postCoreProcess(
+      "add-column", 
+      {
+        baseColumnName: column.name,  
+        newColumnName: "Url of Matched entities", 
+        columnInsertIndex: columnIndex + 1,
+        onError: "set-to-blank"
+      },
+      { expression: '"' + service.view.url + '".replace("{{id}}", cell.recon.match.id)' },
+      { modelsChanged: true },
+      {
+        onDone: function(o) {
+          dismiss();
```

Comment:
I am not sure where this `dismiss` is defined - could it be that it is left over from some other place in the code where you took some inspiration from? I suspect it is not needed here (see the other calls to `Refine.postCoreProcess` above, which do not have any `onDone` callback).

Reply:
Oh yes, you are right, I took it from somewhere and forgot to remove it! doing that asap.